commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -40,21 +45,41 @@
 
   public static final String COMMAND_NAME = "eth2";
 
-  @Spec CommandSpec spec;
+  @Spec CommandSpec commandSpec;
+
+  @CommandLine.Option(
+      names = {"--network"},
+      paramLabel = "<NETWORK>",
+      description =
```

Comment:
This can also be a yaml file path too can't it?

Reply:
yes it can be. Updated description.